src/404.html

```diff
@@ -115,7 +114,7 @@ <h2 class="section-heading">Newsletter</h2>
       </ul>
     </nav>
     <footer>
```

Comment:
Shouldn't this footer tag now be removed and replaced with `{% include footer.html %}`?